src/components/structures/MessagePanel.js

```diff
@@ -376,15 +381,13 @@ module.exports = React.createClass({
                     eventTiles = null;
                 }
 
-                ret.push(
```

Comment:
slightly surprised that this got reformatted - the original layout was much more legible imo